Description: Adds support for the documentation installation
 Installs the HTML documentation share/doc.  When cpp-netlib makes it
 to boost, this package will become obsolete.
 .
 cpp-netlib (0.10.1-3) precise; urgency=low
 .
   * Add documentation package.
Author: Ferdinand Prantl <prantlf@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: vendor
Forwarded: no
Reviewed-By: Ferdinand Prantl <prantlf@gmail.com>
Last-Update: 2013-08-20

--- cpp-netlib-0.10.1.orig/CMakeLists.txt
+++ cpp-netlib-0.10.1/CMakeLists.txt
@@ -35,6 +35,7 @@ if (Boost_FOUND)
     include_directories(${Boost_INCLUDE_DIRS})
     enable_testing()
     add_subdirectory(libs/network/src)
+    add_subdirectory(libs/network/doc)
     add_subdirectory(libs/network/test)
     if (NOT MSVC)
       add_subdirectory(libs/mime/test)
@@ -49,3 +50,4 @@ endif()
 enable_testing()
 
 install(DIRECTORY boost/ DESTINATION include/boost)
+install(FILES index.html DESTINATION share/doc/cpp-netlib-doc/html)
--- /dev/null
+++ cpp-netlib-0.10.1/libs/network/doc/CMakeLists.txt
@@ -0,0 +1,8 @@
+# Copyright (c) Glyn Matthews 2011.
+# Copyright 2011 Dean Michael Berris (dberris@google.com)
+# Copyright 2011 Google, Inc.
+# Distributed under the Boost Software License, Version 1.0.
+#    (See accompanying file LICENSE_1_0.txt or copy at
+#          http://www.boost.org/LICENSE_1_0.txt)
+
+install(DIRECTORY html DESTINATION share/doc/cpp-netlib-doc/html/libs/network/doc)
